Dockerfile

```diff
@@ -1,4 +1,5 @@
 # for testing purposes
+# checkov:skip=CKV_DOCKER_3:Suppress error "Ensure that a user for the container has been created" since this Dockerfile is only used for testing.
```

Comment:
I don't think Checkov supports multiline comments from what I read on the docs (same link as above), but if you're able to find otherwise, that would certainly look nicer.